src/models/ContractEntity.ts

```diff
@@ -2,7 +2,7 @@ import BigNumber from 'bignumber.js';
 import Comparable from '../observer/Comparable';
 
 /**
- * Represents model object.
+ * Represents ContractEntity model object.
```

Comment:
I know the ticket does not mention ContractEntity model and repository, however, believe we should align this one also with others. If you don't want to do it here (as ticket does not indicated this one), please, create a follow ticket.